From 176345d702347b57511a506acf50f5438b516ecc Mon Sep 17 00:00:00 2001
From: yuyue <yuyue@mprc.pku.edu.cn>
Date: Wed, 29 Feb 2012 13:20:44 +0800
Subject: [PATCH 310/641] UniCore64: add root address-cells and size-cells

early_init_dt_scan_memory function needs these two cells for
initializing the base and size of physical memory. Here we set
both the value 1.

Signed-off-by: yuyue <yuyue@mprc.pku.edu.cn>
---
 arch/unicore64/boot/unicore64-fpga.dts |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/unicore64/boot/unicore64-fpga.dts b/arch/unicore64/boot/unicore64-fpga.dts
index 021ac7e..f3acc57 100644
--- a/arch/unicore64/boot/unicore64-fpga.dts
+++ b/arch/unicore64/boot/unicore64-fpga.dts
@@ -1,6 +1,8 @@
 /dts-v1/;
 / {
 	model = "unicore64";
+	#address-cells = <1>;
+	#size-cells = <1>;
 
 	/*
 	 * Memory node.
-- 
1.7.9.5

